carp.common/src/commonMain/kotlin/dk/cachet/carp/common/data/input/element/Text.kt

```diff
@@ -0,0 +1,17 @@
+package dk.cachet.carp.common.data.input.element
+
+import dk.cachet.carp.common.data.input.InputElement
+import kotlinx.serialization.Serializable
+import kotlin.reflect.KClass
+
+
+/**
+ * Text entry by the user.
+ */
+@Serializable
+data class Text( override val name: String ) : InputElement<String>
```

Comment:
Maybe (future) validation rule parameters can be passed as object for the constructor here

Reply:
We can look at this once we start modeling constraints. I don't think we know enough here yet to make informed choices.

I.e., rules might be modular, but might still need to be initialized through configuration properties. Or, rules are polymorphic and serialized alongside as inner objects of `InputElement`.